README.md

```diff
@@ -40,4 +47,13 @@ $ pip install -e .
 ```bash
 $ pip install '.[dev]'
 ```
-(Make sure to include the single quotes)
\ No newline at end of file
+(Make sure to include the single quotes)
+
+### Installing healpy on Mac
+Native prebuilt binaries for healpy on Apple Silicon Macs 
+[do not yet exist](https://healpy.readthedocs.io/en/latest/install.html#binary-installation-with-pip-recommended-for-most-other-python-users), 
+so you may need to run 
+```bash
+$ conda install healpy
```

Comment:
Does the conda install need to happen before the `pip install .`? If so, I'd suggest moving this section to be above the other installation instructions.

Do you need to be using the conda forge channel for this to work?

Reply:
That's a good call--I'd forgotten I'd configured conda forge as my default channel a while ago. Added above.